package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "npm run build:ts && tsc -p test/tsconfig.json && tap --ts \"test/**/*.test.ts\"",
+    "test": "NODE_ENV=test tap --ts \"test/**/*.test.ts\" --coverage-report=text-lcov | tap-nyc --check-coverage && (if [ -f ./test/test.db ]; then rm ./test/test.db; fi)",
```

Comment:
You don't need to complicate thing here, just remove the `if` and `fi`.
You can delete the file anyway! If its there it will get deleted and if not, well it can't do anything else :)
So its okay, just use a simple `rm -rf ...` ant the end of your test command.

Reply:
thanks! edited.

Reply:
thanks! edited.